client/lib/post-normalizer/rule-content-detect-surveys.js

```diff
@@ -0,0 +1,45 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { forEach } from 'lodash';
+import i18n from 'i18n-calypso';
+
+export default function detectSurveys( post, dom ) {
+	if ( ! dom ) {
+		throw new Error( 'this transform must be used as part of withContentDOM' );
+	}
+
+	const surveys = dom.querySelectorAll( '.pd-embed' );
+
+	if ( ! surveys ) {
+		return post;
+	}
+
+	forEach( surveys, survey => {
+		// Get survey details
+		const { domain: surveyDomain, id: surveySlug } = JSON.parse(
+			survey.getAttribute( 'data-settings' )
+		);
+
+		if ( ! surveyDomain || ! surveySlug ) {
+			return;
+		}
+
+		// Construct a survey link
+		const p = document.createElement( 'p' );
+		p.innerHTML =
+			'<a target="_blank" rel="external noopener noreferrer" href="https://' +
+			surveyDomain +
+			surveySlug +
+			'">' +
+			i18n.translate( 'Take our survey' ) +
```

Comment:
Hi! I've found a possible matching string that has already been translated **35** times:
`translate( 'Take our Survey!' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/reader/polldaddy-embeds)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
@bluefuton Should probably use the cap S variant :)